packages/core/copyFiles.js

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env node
+
+/**
+ * Copies files from given path (a glob pattern) to target path.
+ * The prefix part of the glob pattern (before first "*") is replaced with target path.
+ * For example -source "src/x/y/*.scss" with -target "lib/comp"
+ * will copy all matching files from "src/x/y/<rest of path>" to "lib/comp/<rest of path>"
+ * Args:
+ *  - "-s" or "--source" path to scan. It is an glob pattern. Example: 'src\/components\/**\/*.scss'. (escape "\/" added to slashes only to avoid comment block)
+ *  - "-t" or "--target" Target path
+ *  - "-v" verbose. Enable verbose output.
+ */
+
+const { glob } = require('glob');
+var fs = require('fs/promises');
+
+function createPathReplacer(globPath, targetFolder) {
+  const globPrefix = globPath.split('*')[0];
+  const targetPrefix = targetFolder.endsWith('/') ? targetFolder : `${targetFolder}/`;
+  return (path) => {
+    return path.replace(globPrefix, targetPrefix);
+  };
+}
+
+/**
+ * Copies a file with error handling
+ */
+async function copyFile(sourceLocation, targetLocation) {
+  try {
+    await fs.copyFile(sourceLocation, targetLocation);
+  } catch (e) {
+    Promise.resolve(e);
```

Comment:
Hmm... why resolving the Promise here (and not Promise.reject(e))? And hmm... why altogether have a separate function at all but just use the fs.copyfile where this additional function is called? Or is there something I'm missing here 🤔

Reply:
Rejections must be handled, so always resolving prevents catching again.

But, true, could remove the helper / extra function.

Reply:
Everything else seems fine, just this small refactor might be in place. good work 👍

Reply:
Updated.